src/debug.c

```diff
@@ -498,6 +498,9 @@ void debugCommand(client *c) {
 "    Enable or disable the reply buffer resize cron job",
 "CLUSTERLINK KILL <to|from|all> <node-id>",
 "    Kills the link based on the direction to/from (both) with the provided node." ,
+"CLIENT-ENFORCE-REPLY-LIST <0|1>",
+            "When set to 1, it enforces the use of the client reply list directly",
+            "    and avoids using the client's static buffer.",
```

Comment:
Align the output of the DEBUG HELP.

```suggestion
"CLIENT-ENFORCE-REPLY-LIST <0|1>",
"    When set to 1, it enforces the use of the client reply list directly",
"    and avoids using the client's static buffer.",
```